src/resolve.c

```diff
@@ -5038,7 +5038,7 @@ resolve_extension(struct unres_ext *info, struct lys_ext_instance **ext, struct
             /* nothing change */
             break;
         case LYEXT_COMPLEX:
-            tmp_ext = realloc(*ext, ((struct lyext_plugin_complex*)e->plugin)->instance_size);
+            tmp_ext = realloc(*ext, sizeof **ext + ((struct lyext_plugin_complex*)e->plugin)->instance_size);
```

Comment:
Actually it seems correct to me - the `instance_size` is supposed to cover the whole size including the `lys_ext_instance_complex` structure size. But I see a problem in the following memset. A few months ago, there was a change in `lys_ext_instance_complex` structure which doesn't seem to be reflected here. I'll push the fix into the PR branch.